examples/rules/domain.yml

```diff
@@ -38,6 +34,14 @@ actions:
 - utter_greet
 - action_switch_faq
 
+forms:
+- loop_q_form:
+    some_slot:
+    - type: from_entity
+      entity: some_slot
```

Comment:
this is temporary right? we'll need to check that with use team

Reply:
yes of course! It's the same style as in the SDK previously.